src/com/company/Main.java

```
@@ -37,76 +45,83 @@ public static void main(String[] args) {
                     searchContact();
 
                 case 5:
-                    printActions();
+                    questions();
                     break;
 
-//                case 6:
-//                    questions();
-//                    break;
+                case 6:
+                    printOptions();
+                    break;
             }
         }
 
     }
 
+    private static void printOptions() {
+        System.out.println("\nCreate your profile");
+        System.out.println("\t 0 to quit");
+        System.out.println("\t 1 to show persons");
+        System.out.println("\t 2 to add new person");
+        System.out.println("\t 3 to remove person");
+        System.out.println("\t 4 to search person");
+        System.out.println("\t 5 answer our matching questions");
+        System.out.println("\t 6 to show options");
+        System.out.println("Choose your option: ");
+    }
+
     private static void createPerson() {
-        System.out.println("Enter your full name");
-        String fullName = scanner.nextLine();
-        System.out.println("Enter your gender(0 for male, 1 for female)");
+        System.out.println("Enter your name");
+        String name = scanner.nextLine();
+        System.out.println("Enter your gender. ***0 for male <-> 1 for female***");
         int gender = Integer.parseInt(scanner.nextLine());
+        try {
+            if(gender == 0) {
+                Egender male = Egender.MALE;
+                System.out.println(male);
+            }else if(gender == 1) {
+                Egender female = Egender.FEMALE;
+                System.out.println(female);
+            }
+        }catch (InputMismatchException e) {
```

Where is this exception thrown? Research the difference between checked and unchecked exceptions

---

Also dont swallow exceptions, which means that you won't understand where it came from and what caused it. For now use e.printStackTrace() when catching exceptions, later we will discuss the different approaches to exceptions